basaltinc/knapsack
# fix: clear pattern render cache when demos change [KS-167]

To test:

- View a pattern that uses another pattern in one of it's slots
- Go to that sub-pattern, alter the demo data
- Go back to first pattern and note that it contains the new data
